modules/getData.js

```diff
@@ -0,0 +1,39 @@
+const urlGET = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/mwHLZWvmcAi40ernHtsJ/scores/';
+
+const sc = document.getElementById('sc');
+
+const fetchData = () => {
+  fetch(urlGET, {
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error('From GET: Network response was not ok');
+      }
+      return response.json(); // This returns a Promise that resolves with the JSON data
+    })
+    .then((data) => {
+      console.log(data);
+      data.result.forEach((element) => {
```

Comment:
- [ ] [OPTIONAL] You could try to sort the data in descending order so that the highest scores are top of the pile and it is easier to notice the highest scoring players. I believe this will make your project even better 👍